Src/Support/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -60,6 +61,17 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string IncludeGrantedScopes { get; set; }
 
         /// <summary>
+        /// Gets or sets a collection of user defined query parameters to facilitate any not explicitly supported
+        /// by the library which will be included in the resultant authentication URL.
+        /// </summary>
+        /// <remarks>
+        /// The name of this parameter is used only for the constructor and will not end up in the resultant query string.

```

Comment:
I would just clarify it to:
... is used for the &lt;c&gt;RequestParameterAttribute&lt;/c&gt; constructor ...
Make sure that you have no more than 120 chars in a line.